README.md

```diff
@@ -920,6 +924,7 @@ This allows you to check whether the given form element has the specified value.
 It accepts `<input>`, `<select>` and `<textarea>` elements with the exception of
 of `<input type="checkbox">` and `<input type="radio">`, which can be
 meaningfully matched only using [`toHaveFormValue`](#tohaveformvalues).
+`<input type="checkbox>` can also be matched with ['toBeChecked'](#tobechecked).
```

Comment:
What's the point of the single quotes around the tobeChecked link? Maybe you meant to use backticks?